.github/ISSUE_TEMPLATE/1.bug.yml

```diff
@@ -30,7 +32,7 @@ body:
 
         Please provide a link to a live example and an unambiguous set of steps to reproduce this bug. As a starting point, we recommend you [browse the documentation](https://mui.com/x/introduction/), and [select](https://mui.com/static/docs/forking-an-example.png) the closest example to your use case. Or you can use a [basic template](https://mui.com/r/x-issue-template) to build your reproduction case.
```

Comment:
Most of the time people come in with their bug case already known. Having to click on the docs, find an example, fork it, and modify it, is a lot of friction.

I would suggest that we provide from right here a set of starter templates for the different parts of X, this would make it easier for users.

One such example:
 - [Datagrid codesanbox template](https://codesandbox.io/s/mui-datagrid-template-zmcggy?file=/demo.tsx)

Not sure if we want to have direct CSB links or if we want links through mui.com, please let me know what's best.

Reply:
This sounds like a great idea.

It should reduce the pickup time by a fair margin as well, when the case is already known and can be tested against. It might also reduce a ton of misunderstanding (from my experience on past projects this happens quite a lot)

💯

Reply:
Good idea, +1 for direct codesandbox links. How do you see this exactly? 1 link per package? Any preference of codesandbox vs. StackBlitz?

Reply:
Yes one link per package would be nice. I think CSB, we've been using that everywhere.

Reply:
> Most of the time people come in with their bug case already known. Having to click on the docs, find an example, fork it, and modify it, is a lot of friction.
I would suggest that we provide from right here a set of starter templates for the different parts of X, this would make it easier for users.

I struggle to buy this narrative. The current description is:

>  Please provide a link to a live example and an unambiguous set of steps to reproduce this bug. As a starting point, we recommend you [browse the documentation](https://mui.com/x/introduction/), and [select](https://mui.com/static/docs/forking-an-example.png) the closest example to your use case. Or you can use a [basic template](https://mui.com/r/x-issue-template) to build your reproduction case.

The current DX feels like the proposal: you either go with the basic template: https://mui.com/r/x-issue-template, or go with our endless stream of templates that each docs demo is.

Reply:
One use-case that I see quite often when I'm on support duty is the authors open their issue with their existing code pasted literally in the issue body, and we have to ask them for a live reproduction example.

One problem with the current instructions is that it's not super clear/evident that you can fork a codesandbox directly from the docs. It took me weeks to understand that when I started working here, I'm guessing it's possible a lot of people also don't realize it. I think that might be a reason why we often don't have a live example when people open issues.

In any case, I think we could provide both options? A list of direct templates, plus the instructions to the docs. I think we should also be much more explicit, something like `"You are required to provide a codesandbox live example. Fork one of these templates or [docs instructions]"`.

Reply:
I have reworked the message to include both of these, and shortened it a bit to make it more dynamic: [result here](https://github.com/romgrk/mui-test-issue-cleanup/issues/new?assignees=&labels=status%3A+needs+triage&projects=&template=1.bug.yml).